WooCommerce/Classes/ViewRelated/ReusableViews/TitleAndSubtitleAndStatusTableViewCell.swift

```diff
@@ -37,8 +37,7 @@ final class TitleAndSubtitleAndStatusTableViewCell: UITableViewCell, SearchResul
 extension TitleAndSubtitleAndStatusTableViewCell {
     struct ViewModel: Hashable {
         /// A unique ID to avoid duplicated identifier for the view model in diffable datasource
```

Comment:
Could we update the comment to explain why `id` is `var` and when to override the default value? 

Do we have to use our own unique IDs whenever we use diffable data sources? 🤔

Reply:
We should define the ID for more efficient loading when using diffable data sources. I updated the comment in 9a129884a16eaeac052fce04d4985f4f07db258d, thanks!